localstack-core/localstack/config.py

```diff
@@ -433,6 +433,9 @@ def in_docker():
 # When enabled it triggers specialised workflows for the debugging.
 LAMBDA_DEBUG_MODE = is_env_true("LAMBDA_DEBUG_MODE")
```

Comment:
Can we add `LAMBDA_DEBUG_MODE` to the `TRACKED_ENV_VAR` in `localstack-core/localstack/runtime/analytics.py` for telemetry insights?